console_bot/birthdays_per_week.py

```diff
@@ -43,9 +76,24 @@ def prepare_birthday_date(current_date, user_birthday):
 
 if __name__ == "__main__":
     print(get_birthdays_per_week([
-        {"name": "Bill Gates", "birthday": datetime(1981, 2, 11)},
-        {"name": "Steve Jobs", "birthday": datetime(1982, 2, 11)},
-        {"name": "Tim Cook", "birthday": datetime(1991, 2, 12)},
-        {"name": "Jef Bezos", "birthday": datetime(1992, 2, 17)},  # needs to be added to the list next week
+        {"name": "Bill Gates", "birthday": datetime(1981, 3, 14)},
+        {"name": "Steve Jobs", "birthday": datetime(1982, 3, 17)},
+        {"name": "Tim Cook", "birthday": datetime(1991, 3, 17)},
+        {"name": "Jef Bezos", "birthday": datetime(1992, 5, 13)},  # needs to be added to the list next week
         {"name": "Mark Zucherberg", "birthday": datetime(1983, 2, 10)},
     ]))
+
+    print(get_birthdays_in_next_days([
+        {"name": "Bill Gates", "birthday": datetime(1981, 3, 13)},
+        {"name": "Steve Jobs", "birthday": datetime(1982, 3, 14)},
+        {"name": "Tim Cook", "birthday": datetime(1991, 3, 17)},
+        {"name": "Jef Bezos", "birthday": datetime(1992, 5, 13)},  # needs to be added to the list next week
+        {"name": "Mark Zucherberg", "birthday": datetime(1983, 2, 10)},
+    ], 1))
+    print(get_today_birthday([
+        {"name": "Bill Gates", "birthday": datetime(1981, 3, 14)},
+        {"name": "Steve Jobs", "birthday": datetime(1982, 3, 15)},
+        {"name": "Tim Cook", "birthday": datetime(1991, 3, 17)},
+        {"name": "Jef Bezos", "birthday": datetime(1992, 5, 13)},  # needs to be added to the list next week
+        {"name": "Mark Zucherberg", "birthday": datetime(1983, 2, 10)},
+    ]))
```

Comment:
GitHub is asking for a newline at the end of the file. Not sure it's PEP8, but generally people do that